pkg/vault/contracts/authorizer/TimelockAuthorizerMigrator.sol

```diff
@@ -158,6 +153,11 @@ contract TimelockAuthorizerMigrator {
             newAuthorizer.grantPermissions(actionIds, defaultAdmin, wheres);
         }
         newAuthorizer.revokePermissions(actionIds, address(this), wheres);
+
+        // Finally trigger the first step of transferring root ownership over the TimelockAuthorizer to `root`.
+        // Before the migration can be finalized, `root` must call `claimRoot` on the `TimelockAuthorizer`.
+        require(newAuthorizer.canExecute(rootChangeExecutionId), "CANNOT_TRIGGER_ROOT_CHANGE_YET");
+        newAuthorizer.execute(rootChangeExecutionId);
```

Comment:
This is slightly problematic in that we won't be able to do the role migration until the root delay has passed. Ideally the new contract will be complete before the migration is finalized so that addresses and permissions can be checked, etc. before the transfer actually occurs.

Reply:
We'll be able to migrate all the roles except for the last one. Good point though, we should allow the roles to fully migrate without calling `_afterMigrate`